### DISCUSSION TITLE
Errors encountered when doing "make CONFIG=rocket64b2 BOARD=nexys-a7-100t vivado-tcl"

### DISCUSSION THREAD

**Discussion by huahdgithub** [eugene-tarassov/vivado-risc-v#185] [Q&A]
When generating RISC-V RTL in WSL Ubuntu terminal, I got the following error messages:
`$ make CONFIG=rocket64b2 BOARD=nexys-a7-100t vivado-tcl`
`...`
`java.nio.file.FileSystemException: /mnt/g/vivado-risc-v/target/streams/_global/dependencyPositions/_global/streams/update_cache_2.12/output_dsp: Operation not permitted`
`[error]         at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:100)`
`[error]         at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)`
`[error]         at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)`
and so on.
Anybody can help?

**Comment by karnapathak** [comment]
May I know what Java version are you using?
